tests/src/python/test_authsettingswidget.py

```diff
@@ -18,7 +18,7 @@
 import tempfile
 import random
 
-from qgis.core import QgsAuthManager, QgsAuthMethodConfig, QgsNetworkAccessManager, QgsSettings
```

Comment:
No: the test run correctly without it and that class is never used. I'd say that importing QgsAuthManager is from now on completely pointless since you cannot create an instance of that class from python and it doesn't have any public static method. You can access to the auth manager only from QgsApplication